includes/class-wc-payment-token-wcpay-link.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Class WC_Payment_Token_WCPay_SEPA
+ *
+ * @package WooCommerce\Payments
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * WooCommerce Stripe Link Payment Token.
+ *
+ * Representation of a payment token for Link.
+ *
+ * @class    WC_Payment_Token_WCPay_Link
+ */
+class WC_Payment_Token_WCPay_Link extends WC_Payment_Token {
```

Comment:
To me `WC_WCPay_Link_Payment_Token` reads better.

Edit: I now see that you're following the existing convention. That's better